draft-packages/split-button/KaizenDraft/SplitButton/styles.scss

```diff
@@ -243,14 +259,32 @@ $side-padding: 3/4 * $ca-grid;
 .dropdownButtonPrimary {
   composes: buttonReset buttonPrimary;
 
+  --dt-border-left-color-heart: rgba(
+    var(#{$kz-var-id-color-seedling-400-rgb-params}),
+    0.7
+  );
+
   // Making the left | right border white to separate the button with dropdown button
-  border-left-color: $kz-color-white !important; /* stylelint-disable-line declaration-no-important */
+  /* stylelint-disable declaration-no-important */
+  border-left-color: var(
+    --dt-border-left-color-heart,
+    $kz-var-color-white
+  ) !important;
 
   [dir="rtl"] & {
     // equal to add-tint($ca-palette-seedling, 10%). Temporary until we upgrade to Zen (Wisteria)
-    border-left-color: #58b59a !important; /* stylelint-disable-line declaration-no-important */
-    border-left-color: $kz-color-white !important; /* stylelint-disable-line declaration-no-important */
+    border-left-color: #58b59a !important;
+    border-left-color: var(
+      --dt-border-left-color-heart,
+      $kz-var-color-white
+    ) !important;
+  }
+
+  &:active,
+  &:hover {
+    border-left-color: transparent !important;
```

Comment:
Are these rules new?

Reply:
Yes! something that Marc picked up that needed to change.